prow/plugins/trigger/generic-comment.go

```diff
@@ -87,6 +95,24 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 		}
 	}
 
+	pr, err := refGetter.PullRequest()
+	if err != nil {
+		return err
+	}
+
+	// Process "help" comments
+	if testHelpRe.MatchString(gc.Body) {
+		available, err := availablePresubmits(c.GitHubClient, gc.Body, org, repo, pr.Base.Ref, pr.Number, presubmits, c.Logger)
+		if err != nil {
+			return err
+		}
+		resp := fmt.Sprintf("No presubmit jobs available for %s/%s", org, repo)
+		if len(available) > 0 {
+			resp = fmt.Sprintf("The following jobs can be triggered by using their name after a `/test` message: %s", strings.Join(markdownNames(available), ", "))
```

Comment:
comma separated list can get messy, I think we should do a markdown list instead.

Reply:
Thank you for the review. Fixed.